Source/WebInspectorUI/UserInterface/Controllers/CodeMirrorCompletionController.js

```diff
@@ -629,7 +632,10 @@ WI.CodeMirrorCompletionController = class CodeMirrorCompletionController extends
         this._implicitSuffix = suffix !== ":" ? ": " : "";
 
         // Complete property names.
-        return WI.cssManager.propertyNameCompletions.startsWith(this._prefix);
+        if (WI.settings.experimentalUseFuzzyMatchingForCSSCodeCompletion.value)
+            return WI.cssManager.propertyNameCompletions.executeQuery(this._prefix);
+        else
+            return WI.cssManager.propertyNameCompletions.startsWith(this._prefix);
```

Comment:
Style: you dont need the `else` here as it's already implicit because in order to reach this we must not've done the `if` since it would've returned
```js
if (WI.settings.experimentalUseFuzzyMatchingForCSSCodeCompletion.value)
    return WI.cssManager.propertyNameCompletions.executeQuery(this._prefix);
return WI.cssManager.propertyNameCompletions.startsWith(this._prefix);
```